backend/tests/penetration_testing/purchase_order_pentest.py

```
@@ -0,0 +1,247 @@
+This script tests security aspects of the purchase order API endpoints:
+1. File Upload Security Testing
+2. Purchase Order Access Control Testing
+3. Purchase Order Data Injection Testing
+
+API Endpoints:
+- Base URL: http://localhost:3000
+- Purchase Order API: http://localhost:3000/api/purchase-orders
+"""
+
+import requests
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)